agents/check_mk_agent.linux

```diff
@@ -192,9 +192,10 @@ set_up_profiling() {
 preamble_5() {
     unset -v LANG
     if inpath locale && inpath paste; then
-        case "$(locale -a | paste -sd ' ' -)" in
-            *'C.UTF-8'*) LC_ALL="C.UTF-8" ;;
-            *'C.utf8'*) LC_ALL="C.utf8" ;;
+	#     v keep this spaces in place, to catch if string is in first postion
+        case " $(locale -a | paste -sd ' ' -)" in
+            *' C.UTF-8'*) LC_ALL="C.UTF-8" ;;
+            *' C.utf8'*) LC_ALL="C.utf8" ;;
         esac
         LC_ALL="${LC_ALL:-C}"
```

Comment:
As per the discussion, it should be beneficial to move these two lines to beneath the `fi`.  Otherwise, looking good :)